Makefile

```diff
@@ -125,23 +128,28 @@ start-integration: web/src/build/static/app.js bin/goalert bin/psql-lite bin/wai
 jest: node_modules 
 	yarn workspace goalert-web run jest $(JEST_ARGS)
 
-test: node_modules jest
+test: node_modules jest ## Run all unit tests
 	go test -short ./...
 
 force-yarn:
 	yarn install --no-progress --silent --frozen-lockfile --check-files
 
-check: force-yarn generate node_modules
-	# go run ./devtools/ordermigrations -check
-	go vet ./...
-	go run github.com/gordonklaus/ineffassign ./...
-	CGO_ENABLED=0 go run honnef.co/go/tools/cmd/staticcheck ./...
+check: check-go check-js  ## Run all lint checks
+
+check-js: force-yarn generate node_modules
 	yarn run fmt
 	yarn run lint
 	yarn workspaces run check
 	./devtools/ci/tasks/scripts/codecheck.sh
```

Comment:
This isn't a js check, should probably go under the `check` target as it needs to run after both the js and go fmt commands

Reply:
Just put it in the `check` target; another target would just be noise for the sake of noise IMO. `codecheck.sh` is dependent on the other ones being run first, and isn't valid on it's own

Reply:
To clarify: `codecheck.sh` MUST run last, after all linting/formatting/generating, and its task has to depend on both `check-js` and `check-go` running first.

Also, `check-go` also needs to depend on `generate` as that generates code for all languages